src/components/Header/components/Chatting/Chattingroom.tsx

```diff
@@ -169,6 +75,8 @@ function Chattingroom({ roomId, openAuctionInfo }: ChatroomProps) {
     });
   };
 
+  const inputRef = useRef<HTMLInputElement>(null);
```

Comment:
```suggestion
  const inputRef = useRef<HTMLTextAreaElement>(null);
  
  const onKeyDown = (e: KeyboardEvent<HTMLTextAreaElement>) => {
    if (e.key === 'Enter') {
      if (e.shiftKey) {
        return;
      }
      e.preventDefault();

      const message = inputRef.current?.value.trim();
      if (message && inputRef.current) {
        sendMessage(message);
        inputRef.current.value = '';
      }
    }
  };
```
textarea로 변경하면 줄바꿈도 가능해질 것 같아요! 근데 이렇게 되면 중복되는 코드가 조금 있어서 주가적인 함수가 필요할 수도...ㅎ S.chatListWrapper `whiteSpace: 'pre-wrap'` 추가도 필요할 듯 합니다!